### ISSUE TITLE
[Bug][ColumnMenu] Accessibility issues

### ISSUE THREAD

**Issue by filipKovachev** [telerik/kendo-vue#647]
**Describe the bug**
Currently users are not able to open the columnMenu with Tab and open the popup with Enter. It is also not possible to navigate between the checkboxes with tab.

**To Reproduce**
1. Open this example: https://stackblitz.com/edit/przqark8?file=src%2Fmain.vue
2. Notice how the columnMenu is not opened by pressing Enter and the checkboxes are not navigable via Tab

**Expected behavior**
The user should be able to open the columnMenu by pressing Enter once the filter icon is focused and the checkboxes must be navigated using tab.

Reported in Ticket ID: **1672872**

**Comment by VishnuPCMA** [comment]
Hi @filipKovachev 
Can I please get an update regarding this bug ticket. When Can I get the expected result?

**Comment by filipKovachev** [comment]
Still reproducible with version 7.0.2:

https://stackblitz.com/edit/przqark8-xjcqz25c?file=src%2Fmain.vue
